lib/logreader.c

```diff
@@ -349,6 +349,26 @@ log_reader_update_watches(LogReader *self)
     }
 }
 
+static inline gboolean
+log_reader_work_in_progress(LogReader *self)
```

Comment:
If this is only used inside the file, please consider adding a `_` prefix

Reply:
naming follows the other similar, surronding ones here